Command/CreateUserCommand.php

```diff
@@ -108,7 +108,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if (true === $input->getOption('enforce-password-change')) {
             $user->enforcePasswordChange(true);
         }
-        $this->eventDispatcher->dispatch(new UserEvent($user), AdminSecurityEvents::USER_CREATED);
+        $this->eventDispatcher->dispatch(new UserCreatedEvent($user), AdminSecurityEvents::USER_CREATED);
```

Comment:
There is no need for the second argument. We removed it everywhere in `admin-bundle`